test/e2e/specs/editor/plugins/iframed-block.spec.js

```diff
@@ -13,36 +13,17 @@ test.describe( 'Iframed block', () => {
 		await requestUtils.deactivatePlugin( 'gutenberg-test-iframed-block' );
 	} );
 
-	test( 'Should save the changes', async ( { editor, page } ) => {
+	test( 'should load script and dependencies in iframe', async ( {
```

Comment:
The previous test name wasn't descriptive of the actual test and probably a copy/paste error during the migration (#44164).

I've restored the original name.